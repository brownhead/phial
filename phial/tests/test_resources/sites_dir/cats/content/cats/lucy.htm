name: Lucy Lu
...

You are the cat.
