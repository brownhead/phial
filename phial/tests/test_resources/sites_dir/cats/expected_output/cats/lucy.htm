<div>
	<a href="../index.htm">Go Home!</a>
</div>
<h1>Lucy Lu</h1>
<div>
	
<div>
	<b>I am a cat!</b>
</div>

</div>
